packages/tkg-clusterclass-vsphere/bundle/config/upstream/base.yaml

```diff
@@ -2057,9 +2057,64 @@ spec:
             start-service kubelet
             start-service kube-proxy
             start-service antrea-agent
+      - op: add
```

Comment:
Could you hold on this PR to Halifax? It triggers rollout during cc package upgrade.